nf-example-topology/packagevariantset-amf-regional.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: PackageVariantSet
+metadata:
+  name: nf-example-free5g-amf-regional
```

Comment:
But if we change the overall name from "example", I would change the names of all of these resources to align more with the overall package name.

Reply:
Done.